### PR TITLE
[DotNetCoreCLIV2]added .net core 5.0 information

### PR DESCRIPTION
**Pull request by tejasd1990**
**Task name**: <Name of changed or new pipeline task>

**Description**: <Describe your changes here>

**Documentation changes required:** (Y/N) <Please mark if documentation changes are required>

**Added unit tests:** (Y/N) <Please mark if unit tests were added or updated according changes>

**Attached related issue:** (Y/N) <Please add link to related issue here>

**Checklist**:
- [x] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [ ] Checked that applied changes work as expected

### CODE REVIEW

File: Tasks/DotNetCoreCLIV2/Strings/resources.resjson/en-US/resources.resjson [hunk 1]
```diff
@@ -134,6 +134,6 @@
...
+  "loc.messages.NetCore5Update": "Info: Azure Pipelines hosted agents have been updated to contain .Net Core 5.x SDK/Runtime along with the older ones currently lts. Unless you have locked down a SDK version for your project(s), 5.x SDK might be picked up which might have breaking behavior as compared to previous versions. \nYou can learn more about the breaking changes here: \n https://docs.microsoft.com/en-us/dotnet/core/tools/ \n https://docs.microsoft.com/en-us/dotnet/core/compatibility/",
```
Comment (bishal-pdMSFT) [inline Tasks/DotNetCoreCLIV2/Strings/resources.resjson/en-US/resources.resjson:137]: > along with the older ones currently lts

along with older LTS versions

Comment (bishal-pdMSFT) [inline Tasks/DotNetCoreCLIV2/Strings/resources.resjson/en-US/resources.resjson:137]: It would still be useful to keep the troubleshooting link

```
To learn about more such changes and troubleshoot, refer here: https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/build/dotnet-core-cli?view=azure-devops#troubleshooting
```

File: Tasks/Common/utility-common/package-lock.json [hunk 2]
```diff
@@ -107,27 +107,6 @@
       "resolved": "https://registry.npmjs.org/uuid/-/uuid-3.2.1.tgz",
       "integrity": "sha1-EsUou51Y0LkmXZovbw/ovhf/HxQ="
     },
-    "vso-node-api": {
```
Comment (bishal-pdMSFT) [inline Tasks/Common/utility-common/package-lock.json:110]: Please revert this change as it is not at all related

File: Tasks/DotNetCoreCLIV2/dotnetcore.ts [hunk 3]
```diff
@@ -132,7 +132,7 @@ export class dotNetExe {
         }
         if (failedProjects.length > 0) {
             if (this.command === 'publish' && !MessagePrinted) {
-                tl.warning(tl.loc('NetCore3Update'));
+                tl.warning(tl.loc('NetCore5Update'));
```
Comment (bishal-pdMSFT) [inline Tasks/DotNetCoreCLIV2/dotnetcore.ts:135]: I guess this one is no longer required. We had added this as we knew of breaking change in case of publish and wanted to  bubble up that more explicitly as a warning.

I don't think this applies anymore with .Net 5. 
We can also remove that variable `MessagePrinted` then.

**Comment by bishal-pdMSFT** [comment]
How do we control rollout of this change until the updated agents are released

File: Tasks/DotNetCoreCLIV2/Strings/resources.resjson/en-US/resources.resjson (hunk 1 above)
Comment (bishal-pdMSFT) [inline Tasks/DotNetCoreCLIV2/Strings/resources.resjson/en-US/resources.resjson:137]: > Azure Pipelines hosted agents have been updated to contain .Net Core 5.x SDK/Runtime along ...

We are updating Azure Pipelines hosted agents to include .Net Core 5.x SDK/Runtime along ...

**Review by bishal-pdMSFT — APPROVED** [review]

**Comment by tejasd1990** [comment]
>How do we control rollout of this change until the updated agents are released

@bishal-pdMSFT I will merge this PR only when the new agent with dotnet5 is released, and then do a task hotfix
